### PR TITLE
feat: signup page

### PR DESCRIPTION
**Pull request by AdityaRaj0001**
Have added the signup page ,  still have to modify input fields will do them soon.
will implement responsiveness 
will add signin page similar to this through conditional rendering.
will add animations
will implement onboarding steps soon.

### CODE REVIEW

**Comment by S1D007** [comment]
Cool! Can you please share a Preview Link

File: packages/web/app/auth/page.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        </form>
+      </CardContent>
+      <CardFooter className="flex justify-between">
+        <Button variant="outline" onClick={setsigninpage(!signinpage)}>Cancel</Button>
+        <Button>Deploy</Button>
+      </CardFooter>
+    </Card>
+</div>
+    </div>
+
```
Comment (S1D007) [inline packages/web/app/auth/page.tsx]: Please do the Code Formatting too. i will soon setup the Code Formatting

File: packages/web/app/globals.css [hunk 2]
```diff
@@ -2,26 +2,10 @@
...
-body {
-  color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
-}
+#left{
```
Comment (S1D007) [inline packages/web/app/globals.css]: Don't use Pure Css as we are totally commited with Tailwind

File: packages/web/app/auth/page.tsx (hunk 1 above)
Comment (AdityaRaj0001) [reply packages/web/app/auth/page.tsx]: oki

**Review by S1D007 — CHANGES_REQUESTED** [review]
Please do the following review's. @AdityaRaj0001

File: packages/web/app/globals.css (hunk 2 above)
Comment (AdityaRaj0001) [reply packages/web/app/globals.css]: ohk got it

**Pull request merged**